packages/mapviewer/src/utils/components/WarningButton.vue

```diff
@@ -0,0 +1,74 @@
+<script setup>
+import tippy from 'tippy.js'
+import { computed, onBeforeUnmount, onMounted, ref, toRefs, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useStore } from 'vuex'
+
+import WarningMessage from '@/utils/WarningMessage.class'
+
+const props = defineProps({
+    compact: {
+        type: Boolean,
+        required: true,
+    },
+    warningMessage: {
+        type: WarningMessage,
+        required: true,
+    },
+})
+const { compact, warningMessage } = toRefs(props)
+const warningButton = ref(null)
+
+const i18n = useI18n()
+const store = useStore()
+
+const lang = computed(() => store.state.i18n.lang)
+const translatedMessage = computed(() =>
+    i18n.t(warningMessage.value.msg, warningMessage.value.params)
+)
+
+let tippyInstance = null
+
+watch(lang, () => tippyInstance?.setContent(translatedMessage.value))
+
+onMounted(() => {
+    tippyInstance = tippy(warningButton.value, {
+        theme: 'warning',
+        content: translatedMessage.value,
+        arrow: true,
+        placement: 'top',
+        touch: false,
+        hideOnClick: false,
+        onCreate: (instance) => {
+            const dataCy = instance.reference.getAttribute('data-cy')
+            if (dataCy) {
+                instance.popper.setAttribute('data-cy', `tippy-${dataCy}`)
+            }
+        },
+    })
```

Comment:
can you rebase your work on https://github.com/geoadmin/web-mapviewer/pull/1248 and directly use the new FloatingUI component instead?

Reply:
seems like a good idea